mappings/net/minecraft/client/render/entity/EntityRendererExplodingWitherSkull.mapping

```diff
@@ -1,3 +1,9 @@
 CLASS none/bvb net/minecraft/client/render/entity/EntityRendererExplodingWitherSkull
+	FIELD a WITHER_INVINSABLE_SKIN Lnone/kp;
```

Comment:
`INVINCIBLE` - Also this is `ExplodingWitherSkull` so either the class or the field is named incorrectly.

Is that the same name as we're using in `EntityWither`?